java/Oleksii-Riabov/src/com/zoolatech/lecture4/tasks/_2/Tree.java

```diff
@@ -0,0 +1,50 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class Tree {
+
+    class Node {
+    int value;
+    Node leftNode;
+    Node rightNode;
+
+    Node(int value) {
+        this.value = value;
+        rightNode = null;
+        leftNode = null;
+    }
+}
+
+    Node root;
+
+    private Node addRecursive(Node current, int value) {
```

Comment:
prefer to define methods in order they are used: in current case, method `add` uses method `addRecursive`, so it's better to define them in the same order

Reply:
Already change order of methods in this class for better readability.